sdk/communication/tests.yml

```diff
@@ -28,15 +28,33 @@ extends:
       SKIP_PHONENUMBER_LIVE_TESTS: TRUE
       TEST_PACKAGES_ENABLED: ${{ parameters.TestPackagesEnabled }}
     PostSteps:
+        - script: >-
+            dotnet test eng/service.proj --filter "(TestCategory!=Manually) & (TestCategory!=Live)" --framework $(TestTargetFramework)
+            --logger "trx;LogFileName=$(TestTargetFramework).trx" --logger:"console;verbosity=normal"
+            /p:SDKType=${{ parameters.SDKType }}
+            /p:ServiceDirectory=${{ parameters.ServiceToTest }}
+            /p:IncludeSrc=false /p:IncludeSamples=false /p:IncludePerf=false /p:IncludeStress=false
+            /p:RunApiCompat=false /p:InheritDocEnabled=false
+            /p:Configuration=$(BuildConfiguration) $(ConvertToProjectReferenceOption)
+            /p:CollectCoverage=$(CollectCoverage) /p:CodeCoverageDirectory=$(Build.SourcesDirectory)\sdk\${{parameters.ServiceDirectory}}
+          displayName: "Build & Test ($(TestTargetFramework))"
+        - task: PublishTestResults@2
+          condition: always()
+          displayName: "Publish Results ($(TestTargetFramework))"
+          inputs:
+            testResultsFiles: "**/$(TestTargetFramework)*.trx"
+            testRunTitle: "$(OSName) $(TestTargetFramework)"
+            testResultsFormat: "VSTest"
+            mergeTestResults: true
         - task: Palmmedia.reportgenerator.reportgenerator-build-release-task.reportgenerator@4
           condition: and(succeededOrFailed(), eq(variables['CollectCoverage'], 'true'), eq(variables['coverage.collected'], 'true'))
           displayName: Generate Code Coverage Reports
           inputs:
-            reports: $(Build.SourcesDirectory)\sdk\${{parameters.ServiceDirectory}}\**\coverage.cobertura.xml
-            targetdir: $(Build.ArtifactStagingDirectory)\coverage
-            reporttypes: Cobertura
-            filefilters: +$(Build.SourcesDirectory)\sdk\${{parameters.ServiceDirectory}}\**
-            verbosity: Verbose
+          reports: $(Build.SourcesDirectory)\sdk\${{parameters.ServiceDirectory}}\**\coverage.cobertura.xml
+          targetdir: $(Build.ArtifactStagingDirectory)\coverage
+          reporttypes: Cobertura
+          filefilters: +$(Build.SourcesDirectory)\sdk\${{parameters.ServiceDirectory}}\**
+          verbosity: Verbose
```

Comment:
This doesn't look right. These are all inputs. Putting them at the same level as "inputs:" in YAML removes them as actual inputs.